junit-jupiter-params/junit-jupiter-params.gradle.kts

```diff
@@ -33,20 +39,23 @@ tasks {
 			""")
 		}
 	}
-}
-
-tasks {
 	shadowJar {
 		relocate("com.univocity", "org.junit.jupiter.params.shadow.com.univocity")
 		from(projectDir) {
 			include("LICENSE-univocity-parsers.md")
 			into("META-INF")
 		}
 	}
-	compileModule {
+	compileJava {
 		options.compilerArgs.addAll(listOf(
 			"--add-modules", "univocity.parsers",
 			"--add-reads", "${javaModuleName}=univocity.parsers"
 		))
 	}
+	javadoc {
+		(options as StandardJavadocDocletOptions).apply {
+			addStringOption("-add-modules", "univocity.parsers")
+			addStringOption("-add-reads", "${javaModuleName}=univocity.parsers")
```

Comment:
:speech_balloon: Javadoc for individual subprojects now also uses the module path instead of the classpath and hence needs these options as well.